stencil-library/src/components/payment-method-form/payment-method-form.tsx

```diff
@@ -60,7 +60,7 @@ export class PaymentMethodForm {
   }
 
   private postMessage(eventType: string, payload?: any) {
-    if (this.iframeElement) {
+    if (this.iframeElement && this.iframeElement.contentWindow) {
```

Comment:
Hi @jakemer 
I have a suggestion, instead of using an if we could use optional chaining, which is slightly better to read. WDYT?
`this?.iframeElement?.contentWindow?.postMessage({ eventType: eventType, ...payload }, '*');`

Reply:
@msencenb Yep! It does get transformed in the build pipeline. There is also a `webcomponents.cjs.js` that you can use instead which is common JS